src/CleanArchitecture.Web/Startup.cs

```diff
@@ -33,13 +30,14 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                 options.CheckConsentNeeded = context => true;
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
-            // TODO: Add DbContext and IOC
-            string dbName = Guid.NewGuid().ToString();
 
-            services.AddDbContext<AppDbContext>(options =>
-                options.UseSqlite("Data Source=database.sqlite")); // will be created in web project root
-//                options.UseInMemoryDatabase(dbName));
-                //options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection")));
+            // TODO: Not sure where to call this database init now that we don't have access to AppDbContext
```

Comment:
What to do for lack of access to the `AppDbContext` type?

Reply:
Use reflection to grab the infra assembly and instantiate it from there?

Reply:
Use reflection to grab the infra assembly and instantiate it from there?